pkg/gce-pd-csi-driver/node.go

```diff
@@ -327,6 +332,41 @@ func (ns *GCENodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStage
 	return &csi.NodeStageVolumeResponse{}, nil
 }
 
+func (ns *GCENodeServer) volumeCompatible(mountedPath, devicePath string, volumeCapability *csi.VolumeCapability) error {
+	// Part 1: Check that volume mounted at mountedPath is the same as one at
+	// devicePath
+	devicePathDev, err := filepath.EvalSymlinks(devicePath)
+	if err != nil {
+		return fmt.Errorf("failed to find backing disk for devicePath %s: %v", devicePath, err)
+	}
+
+	mountedPathDevBytes, err := ns.Mounter.Exec.Command("sh", "-c", fmt.Sprintf("df -P | awk '$6==\"%s\"{print $1}'", mountedPath)).CombinedOutput()
```

Comment:
Can you add a comment that shows the syntax of the line returned and the output post processing?